src/controllers/login.js

```diff
@@ -0,0 +1,44 @@
+import UserService from '../services/UserService';
+import { isPasswordTrue } from '../utils/paaswordDecrypt';
```

Comment:
can you rename this file `../utils/paaswordDecrypt` to `../utils/passwordDecrypt`. It's just a little misspell there :)

Reply:
done

Reply:
Great @descholar-ceo, would you push your changes then